tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+	"compilerOptions": {
+		"allowJs": true,
+		"declaration": true,
+		"emitDeclarationOnly": true,
+		"esModuleInterop": true,
+		"forceConsistentCasingInFileNames": true,
+		"skipLibCheck": true,
+		"strict": true,
+		"baseUrl": ".",
+		"moduleResolution": "node",
+		"module": "ES2020",
+		"target": "ES2020",
+		"paths": {
+			"~/*": ["src/*"]
+		},
+		"outDir": "./dist",
+		"declarationDir": "./dist/types",
```

Comment:
Just double checking: does anything break without this? We didn’t need it with jsconfig so just wondering. It does align with the build output but could get out of sync with `astro.config.mjs` in theory, so I wouldn’t include it unless we definitely need it.

Reply:
Yes, if you remove these, `tsc` breaks. It would overwrite the source files and errors out.